packages/salesforcedx-vscode-core/src/commands/deploySourcePath.ts

```diff
@@ -19,8 +20,9 @@ import { CompositePostconditionChecker } from './util/compositePostconditionChec
 import { TimestampConflictChecker } from './util/timestampConflictChecker';
 
 export class LibraryDeploySourcePathExecutor extends DeployExecutor<string[]> {
-  constructor() {
+  constructor(showChannelOutput: boolean = true) {
```

Comment:
This is one area I debated about. 1) passing in the indicator to show the output panel as a constructor argument or 2) creating a separate instance method like `setShowChannelOutput()` called after LibraryDeploySourcePathExecutor is instantiated.

Option 2 would make the ` new SfCommandlet<string[]>(` command below a little more complicated as you'd have to instantiate ` LibraryDeploySourcePathExecutor` call the method to set the output, then pass the object to the `SfCommandlet` constructor.

Open to thoughts on either approach.